packages/core/src/render3/i18n.ts

```diff
@@ -932,68 +950,98 @@ function readUpdateOpCodes(
   }
 }
 
-function icuUpdateCase(
-    tView: TView, lView: LView, tIcuIndex: number, nodeIndex: number, bindingsStartIndex: number,
-    tIcus: TIcu[], caseCreated: boolean) {
-  const tIcu = tIcus[tIcuIndex];
-  const icuTNode = getTNode(tView, nodeIndex) as TIcuContainerNode;
-  if (icuTNode.activeCaseIndex !== null) {
-    readUpdateOpCodes(
-        tIcu.update[icuTNode.activeCaseIndex], tIcus, bindingsStartIndex, changeMask, tView, lView,
-        caseCreated);
+/**
+ * Apply OpCodes associated with updating an existing ICU.
+ *
+ * @param tView Current `TView`
+ * @param tIcus If ICUs present than this contain them.
```

Comment:
I believe in case of `applyIcuUpdateCase`, ICUs are always present.
```suggestion
 * @param tIcus ICUs active at this location.
```

Reply:
fixed